Flatten exercise: add extra instructions (#277)
exercism/jq: exercises/practice/flatten-array/.docs/instructions.append.md

# ignore

## jq-specific Instructions

> There is a builtin [`flatten`][flatten] function that can do most of the work for you.
> For maximum learning, try to implement this yourself without using the builtin function.

Reviewer: ```suggestion
There is a builtin [`flatten`][flatten] function that largely solves this exercise.
Try to solve this yourself without using the builtin `flatten` function to get the most out of this exercise.
```

<details><summary>Click here for hints:</summary>

> * Using a recursive function may be useful.
>   Learn more about recursion in the [Recursion learning document][recur].
> * The [`type`][type] function will help.

Reviewer: ```suggestion
* A recursive function can be useful.
  Learn more about recursion in the [Recursion lesson][recur].
* The [`type`][type] function can help.
```
